app/build.gradle

```diff
@@ -64,12 +72,21 @@ dependencies {
     annotationProcessor("androidx.room:room-compiler:$room_version")
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4"
     testImplementation("androidx.arch.core:core-testing:2.2.0")
-    def lifecycle_version = "2.5.1"
+    def lifecycle_version = "2.6.0"
     implementation "androidx.lifecycle:lifecycle-livedata-ktx:$lifecycle_version"
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycle_version"
     implementation 'com.squareup.retrofit2:retrofit:2.9.0'
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
     implementation 'com.squareup.okhttp3:okhttp:5.0.0-alpha.1'
     testImplementation("com.squareup.okhttp3:mockwebserver:4.10.0")
     testImplementation "org.mockito:mockito-core:3.12.4"
+    annotationProcessor("com.github.bumptech.glide:compiler:4.15.0")
```

Comment:
Why annotationProcessor is used?

Reply:
This was required to build a glideappmodule.

Error message : Failed to find GeneratedAppGlideModule. You should include an annotationProcessor compile dependency on com.github.bumptech.glide:compiler in your application and a @GlideModule annotated AppGlideModule implementation or LibraryGlideModules will be silently ignored
Not implemented it yet.

Reply:
We can use kapt since we are mostly working with kotlin right